activemodel/lib/active_model/secure_password.rb

```diff
@@ -93,10 +93,10 @@ def initialize(attribute)
         attr_reader attribute
 
         define_method("#{attribute}=") do |unencrypted_password|
-          if unencrypted_password.nil?
+          if unencrypted_password.nil? || unencrypted_password.empty?
```

Comment:
Ah you didn't use `unencrypted_password.blank?` after all.
```suggestion
          if unencrypted_password.blank?
```

Reply:
oops, I was updated with `blank?` now :)